intertidal/elevation.py

```diff
@@ -1259,7 +1259,7 @@ def intertidal_cli(
         )
 
         # Add coastal mask output layer
-        ds["qa_coastal_mask"] = coastal_mask
+        ds["qa_coastal_connectivity"] = conn_mask.where(conn_mask<65535,drop=True)
```

Comment:
I don't think `drop=True` is required here; we're just replacing values with `nodata`, not reshaping the array or dropping pixels

```suggestion
        ds["qa_coastal_connectivity"] = conn_mask.where(conn_mask < 65535)
```